browser/js/common/directives/corners/corners.js

```diff
@@ -38,15 +52,88 @@ app.directive('corners', function() {
             var canvas = document.getElementById("canvas");
             var context = canvas.getContext("2d");
 
+
+            //Config boxes and letters.
+            scope.box = [0, 0, 0, 0, 0, 0, 0, 0]
+            scope.TLBox = ["A", "B", "C", "D", "E", ""];
+            scope.TRBox = ["F", "G", "H", "I", "J", ""];
+            scope.MMBox = ["K", "L", "M", "N", "O", ""];
+            scope.BLBox = ["P", "Q", "R", "S", "T", ""];
+            scope.BRBox = ["U", "V", "W", "X", "Y", "Z"];
+
```

Comment:
How about creating a admin page to create this array? Possibly move it to a 2D array in future? Check out the array that I made in grid.js